recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "0.41.1" %}
+{% set version = "0.41.2" %}
```

Comment:
Not a blocker, but it would be nice for future reference to add a comment in the PR's description saying the version is `0.41.2` even though it does not match the tag.